spec/v0.1/schema.json

```diff
@@ -253,6 +268,12 @@
         },
         "blocks": {
           "$ref": "#/$defs/resource_blocks"
+        },
+        "description": {
+          "type": "string"
+        },
+        "markdown_description": {
+          "type": "string"
```

Comment:
Can you add the mapped `deprecation_message` and `version` from the Go bindings here?